app/models/org_space_policy.rb

```diff
@@ -0,0 +1,75 @@
+require 'conjur_client'
+
+class OrgSpacePolicy
+  class OrgPolicyNotFound < RuntimeError
+  end
+
+  class SpacePolicyNotFound < RuntimeError
+  end
+
+  class SpaceLayerNotFound < RuntimeError
+  end
+
+  class << self
+    def ensure_exists(org_id, space_id)
+      OrgSpacePolicy.new(org_id, space_id).ensure_exists
+    end
+  end
+
+  def initialize(org_id, space_id)
+    @org_id = org_id
+    @space_id = space_id
+  end
+
+  def ensure_exists
+    ensure_org_policy
+    ensure_space_policy
+    ensure_space_layer
```

Comment:
any reason to not also `ensure_org_layer`?

Reply:
The service broker, as coded, doesn't interact with the org layer directly on bind, so we don't need it to ensure it exist as a precondition.

That's the only reason I don't check it. If you would prefer it to also error if it's not present, I can do so.

Reply:
We should at least have an automated test in your provision PR to ensure that the org layer exists and the host is (by inheritance) a member of it. If we are going to advise customers they can entitle the org policy layer, I'd like to have tests to ensure that will work.

Can you please add a note on this in the provision issue and ensure it gets implemented there?

Reply:
Sure thing, will do!